hack/make.sh

```diff
@@ -77,7 +77,7 @@ if [ "$AUTO_GOPATH" ]; then
 	export GOPATH="${PWD}/.gopath"
 fi
 
-if [ ! "$GOPATH" ]; then
+if [ ! "$GOPATH" ] && [ ! "$(go env GOPATH)" ]; then
```

Comment:
Silly question; wouldn't the first check be redundant? (i.e., wouldn't `go env GOPATH` reflect `GOPATH` (if set)?)

Reply:
LOL, actually wondering now if it's possible to _not_ have `GOPATH`;

```bash
root@f0c269d1aafd:/go# printenv GOPATH
/go
root@f0c269d1aafd:/go# go env GOPATH
/go
root@f0c269d1aafd:/go# export GOPATH=/foo/bar
root@f0c269d1aafd:/go# go env GOPATH
/foo/bar
root@f0c269d1aafd:/go# unset GOPATH
root@f0c269d1aafd:/go# go env GOPATH
/root/go
```